src/common/addlayers/ServerService.js

```diff
@@ -661,8 +666,9 @@ var SERVER_SERVICE_USE_PROXY = true;
     };
 
     this.addSearchResultsForHyper = function(server, filterOptions) {
-      var searchUrl = 'http://geoshape.geointservices.io/search/hypermap/_search?';
+      var searchUrl = service_.catalogList[0].url + '_search?';
       if (filterOptions !== null) {
+        searchUrl = service_.catalogList[filterOptions.catalogKey].url + '_search?';
         searchUrl = service_.applyESFilter(searchUrl, filterOptions);
       }
```

Comment:
Hi @milafrerichs, I have some problems with the test, and I don't understand why, can you help me?